kart/commit.py

```diff
@@ -51,7 +95,18 @@
     help=(
         "Usually recording a commit that has the exact same tree as its sole "
         "parent commit is a mistake, and the command prevents you from making "
-        "such a commit. This option bypasses the safety"
+        "such a commit. This option bypasses the safety."
+    ),
+)
+@click.option(
+    "--allow-pk-conflicts",
+    is_flag=True,
+    default=False,
+    help=(
+        "Usually, inserting some new features into the working copy that have the same primary key values as existing "
+        "features that are not in the working copy because they do not match the spatial filter is a mistake - if "
+        "these were committed, it could accidentally overwrite existing features you were unaware of. This option "
+        "bypasses the safety."
```

Comment:
the first clause here is a marathon, I suggest breaking it up a bit
```suggestion
        "Usually, it is a mistake to check features into the working copy that have the same primary key "
        "as something that is outside the spatial filter. If such features were committed, it could "
        "accidentally overwrite existing features you were unaware of. So by default, this is not 
        "allowed. This option bypasses the safety."
```

Reply:
Done